File: code.c

// Auteur: ProgKIT
// Date: 10-03-2023


#include <stdio.h>

#define MIN 1
#define MAX 100
int main()
{
    
    int nbAleatoire, nbEntrer;
    char rep = 'o';
    
    
    //while(rep == 'o') {
        printf("***************************************\n");
        printf("*\tJEU DE DEEVINETTE \n");
        printf("***************************************\n");
        
        nbAleatoire = rand() % (MAX - MIN + 1) + MIN;
        
        printf("\nQuel est le nombre entre (%d et %d):  ", MIN, MAX);
        scanf("%d", &nbEntrer);
        
        if(nbAleatoire < nbEntrer){
            printf("\nPLUS GRAND !!!");
        } else if (nbAleatoire > nbEntrer){
            printf("\nPLUS PETIT !!!");   
        } else {
            printf("\nBRAVO !!!");
        }
        fflush(stdin);
    //    printf("\nVoulez-vous recommencer?(o/n): ");
    //    rep = getchar();
    //}
    
    //printf("\n\nBonne journee!!!\n");

    return 0;
}

File: code.java
// Auteur: ProgKIT
// Date: 10-03-2023

import java.util.Scanner;
import java.util.Random;

public class Main {

    public static final int MIN = 1;
    public static final int MAX = 100;
    
    public static void main(String[] args) {
        
        int nbAleatoire, nbEntrer;
       // char rep = 'o';
        Scanner scanner = new Scanner(System.in);
        Random random = new Random();
        
       // while(rep == 'o') {
            System.out.println("***************************************");
            System.out.println("*\tJEU DE DEEVINETTE ");
            System.out.println("***************************************");
            
            nbAleatoire = random.nextInt((MAX - MIN) + 1) + MIN;
            
            System.out.print("\nQuel est le nombre entre (" + MIN + " et " + MAX + "): ");
            nbEntrer = scanner.nextInt();
            
            if(nbAleatoire < nbEntrer){
                System.out.println("\nPLUS GRAND !!!");
            } else if (nbAleatoire > nbEntrer){
                System.out.println("\nPLUS PETIT !!!");   
            } else {
                System.out.println("\nBRAVO !!!");
            }
            
        //    scanner.nextLine();
            
        //    System.out.print("\nVoulez-vous recommencer? (o/n): ");
            //rep = scanner.nextLine().charAt(0);
        //}
        
        //System.out.println("\n\nBonne journée !!!");
        
        
    }
}